File: web/lang.cs
<?php
// $Id$

// This file contains PHP code that specifies language specific strings
// The default strings come from lang.en, and anything in a locale
// specific file will overwrite the default. This is a Czech file.
//
// Translations provided by: "SmEjDiL" <malyl@col.cz>, 
//   "David Krotil" <David.Krotil@mu-sokolov.cz>
//
// This file is PHP code. Treat it as such.

// The charset to use in "Content-type" header
$vocab["charset"]            = "iso-8859-2";

// Used in style.inc
$vocab["mrbs"]               = "MRBS - Rezervaèní systém";

// Used in functions.inc
$vocab["report"]             = "Výpis";
$vocab["admin"]              = "Administrátor";
$vocab["help"]               = "Pomoc";
$vocab["search"]             = "Hledat";
$vocab["not_php3"]             = "<H1>UPOZORNÌNÍ: Toto zøejmì není funkèní s PHP3</H1>";

// Used in day.php
$vocab["bookingsfor"]        = "Objednáno pro";
$vocab["bookingsforpost"]    = ""; // Goes after the date
$vocab["areas"]              = "Oblasti";
$vocab["daybefore"]          = "Den vzad";
$vocab["dayafter"]           = "Den vpøed";
$vocab["gototoday"]          = "Dnes";
$vocab["goto"]               = "Pøejít na";
$vocab["highlight_line"]     = "Oznaète tuto øádku";
$vocab["click_to_reserve"]   = "Klepnìte na buòku, aby jste provedli rezervaci.";

// Used in trailer.inc
$vocab["viewday"]            = "Dny";
$vocab["viewweek"]           = "Týdny";
$vocab["viewmonth"]          = "Mìsíce ";
$vocab["ppreview"]           = "Pro tisk";

// Used in edit_entry.php
$vocab["addentry"]           = "Pøidat záznam";
$vocab["editentry"]          = "Editovat záznam";
$vocab["editseries"]         = "Editovat sérii";
$vocab["namebooker"]         = "Popis instrukce";
$vocab["fulldescription"]    = "Celkový popis:<br>&nbsp;&nbsp;(Poèet cestujících,<br>&nbsp;&nbsp;Obsazeno/Volná místa atd)";
$vocab["date"]               = "Datum";
$vocab["start_date"]         = "Zaèátek";
$vocab["end_date"]           = "Konec";
$vocab["time"]               = "Èas";
$vocab["period"]             = "Perioda";
$vocab["duration"]           = "Doba trvání";
$vocab["seconds"]            = "sekundy";
$vocab["minutes"]            = "minuty";
$vocab["hours"]              = "hodiny";
$vocab["days"]               = "dny";
$vocab["weeks"]              = "víkendy";
$vocab["years"]              = "roky";
$vocab["periods"]            = "period";
$vocab["all_day"]            = "Všechny dny";
$vocab["type"]               = "Typ";
$vocab["internal"]           = "Volná místa";
$vocab["external"]           = "Obsazeno";
$vocab["save"]               = "Uložit";
$vocab["rep_type"]           = "Typ opakování";
$vocab["rep_type_0"]         = "Nikdy";
$vocab["rep_type_1"]         = "Dennì";
$vocab["rep_type_2"]         = "Týdnì";
$vocab["rep_type_3"]         = "Mìsíènì";
$vocab["rep_type_4"]         = "Roènì";
$vocab["rep_type_5"]         = "Mìsíènì, jednou za mìsíc";
$vocab["rep_type_6"]         = "n-týdnù";
$vocab["rep_end_date"]       = "Konec opakování";
$vocab["rep_rep_day"]        = "Opakovat v den";
$vocab["rep_for_weekly"]     = "(pro (n-)týdnù)";
$vocab["rep_freq"]           = "Frekvence";
$vocab["rep_num_weeks"]      = "Èislo týdne";
$vocab["rep_for_nweekly"]    = "(pro n-týdnù)";
$vocab["ctrl_click"]         = "Užít CTRL pro výbìr více místností";
$vocab["entryid"]            = "Vstupní ID ";
$vocab["repeat_id"]          = "ID pro opakování"; 
$vocab["you_have_not_entered"] = "Nevložil jste";
$vocab["you_have_not_selected"] = "Nevybral jste";
$vocab["valid_room"]         = "prostøedek.";
$vocab["valid_time_of_day"]  = "platný èasový úsek dne.";
$vocab["brief_description"]  = "Krátký popis.";
$vocab["useful_n-weekly_value"] = "použitelná x-týdenní hodnota.";

// Used in view_entry.php
$vocab["description"]        = "Popis";
$vocab["room"]               = "Místnost";
$vocab["createdby"]          = "Vytvoøil uživatel";
$vocab["lastupdate"]         = "Poslední zmìna";
$vocab["deleteentry"]        = "Smazat záznam";
$vocab["deleteseries"]       = "Smazat sérii";
$vocab["confirmdel"]         = "Jste si jistý\\nsmazáním tohoto záznamu?\\n\\n";
$vocab["returnprev"]         = "Návrat na pøedchozí stránku";
$vocab["invalid_entry_id"]   = "Špatné ID záznamu.";
$vocab["invalid_series_id"]  = "Špatné ID skupiny.";

// Used in edit_entry_handler.php
$vocab["error"]              = "Chyba";
$vocab["sched_conflict"]     = "Konflikt pøi plánování";
$vocab["conflict"]           = "Nová rezervace je v konfliktu s jiným záznamem";
$vocab["too_may_entrys"]     = "Vybraná volba byla vytvoøena pro jiné záznamy.<br>Prosím vyberte jinou volbu!";
$vocab["returncal"]          = "Návrat do kalendáøe";
$vocab["failed_to_acquire"]  = "Chyba výhradního pøístupu do databáze"; 
$vocab["mail_subject_entry"] = $mail["subject"];
$vocab["mail_body_new_entry"] = $mail["new_entry"];
$vocab["mail_body_del_entry"] = $mail["deleted_entry"];
$vocab["mail_body_changed_entry"] = $mail["changed_entry"];
$vocab["mail_subject_delete"] = $mail["subject_delete"];

// Authentication stuff
$vocab["accessdenied"]       = "Pøístup zamítnut";
$vocab["norights"]           = "Nemáte pøístupové právo pro zmìnu této položky.";
$vocab["please_login"]       = "Prosím, pøihlašte se";
$vocab["user_name"]          = "Jméno";
$vocab["user_password"]      = "Heslo";
$vocab["unknown_user"]       = "Neznámý uživatel";
$vocab["you_are"]            = "Jste";
$vocab["login"]              = "Pøihlásit se";
$vocab["logoff"]             = "Odhlásit se";

// Authentication database
$vocab["user_list"]          = "Seznam uživatelù";
$vocab["edit_user"]          = "Editovat uživatele";
$vocab["delete_user"]        = "Smazat tohoto uživatele";
//$vocab["user_name"]         = Use the same as above, for consistency.
//$vocab["user_password"]     = Use the same as above, for consistency.
$vocab["user_email"]         = "Emailová adresa";
$vocab["password_twice"]     = "Pokud chcete zmìnit heslo, prosím napište ho dvakrát";
$vocab["passwords_not_eq"]   = "Chyba: Vložená hesla se neshodují.";
$vocab["add_new_user"]       = "Pøidat nového uživatele";
$vocab["rights"]             = "Práva";
$vocab["action"]             = "Akce";
$vocab["user"]               = "Uživatel";
$vocab["administrator"]      = "Administrátor";
$vocab["unknown"]            = "Neznámý";
$vocab["ok"]                 = "Ano";
$vocab["show_my_entries"]    = "Klepnout pro zobrazání všech nadcházejících záznamù";

// Used in search.php
$vocab["invalid_search"]     = "Prázdný nebo neplatný hledaný øetìzec.";
$vocab["search_results"]     = "Výsledek hledání pro";
$vocab["nothing_found"]      = "Nic nenalezeno";
$vocab["records"]            = "Záznam";
$vocab["through"]            = " skrze ";
$vocab["of"]                 = " o ";
$vocab["previous"]           = "Pøedchozi";
$vocab["next"]               = "Další";
$vocab["entry"]              = "Záznam";
$vocab["view"]               = "Náhled";
$vocab["advanced_search"]    = "Rozšíøené hledání";
$vocab["search_button"]      = "Hledat";
$vocab["search_for"]         = "Hledat co";
$vocab["from"]               = "Od";

// Used in report.php
$vocab["report_on"]          = "Výpis setkání";
$vocab["report_start"]       = "Výpis zaèátkù";
$vocab["report_end"]         = "Výpis koncù";
$vocab["match_area"]         = "Hledaná oblast";
$vocab["match_room"]         = "Hledaná místnost";
$vocab["match_type"]         = "Hledaný typ";
$vocab["ctrl_click_type"]    = "Užít CTRL pro výbìr více typù";
$vocab["match_entry"]        = "Hledat v popisu";
$vocab["match_descr"]        = "Hledat v celém popisu";
$vocab["include"]            = "Zahrnovat";
$vocab["report_only"]        = "Jen výpis";
$vocab["summary_only"]       = "Jen pøehled";
$vocab["report_and_summary"] = "Výpis a pøehled";
$vocab["summarize_by"]       = "Pøehled od";
$vocab["sum_by_descrip"]     = "Popis instrukce";
$vocab["sum_by_creator"]     = "Tvùrce";
$vocab["entry_found"]        = "nalezeno";
$vocab["entries_found"]      = "nalezeno";
$vocab["summary_header"]     = "Pøehled  (záznamu) hodiny";
$vocab["summary_header_per"] = "Pøehled  (záznamu) periody";
$vocab["total"]              = "Celkem";
$vocab["submitquery"]        = "Vytvoøit sestavu";
$vocab["sort_rep"]           = "Seøadit výpis podle";
$vocab["sort_rep_time"]      = "Výchozí den/èas";
$vocab["rep_dsp"]            = "Zobrazit ve výpisu";
$vocab["rep_dsp_dur"]        = "Trvání";
$vocab["rep_dsp_end"]        = "Èas ukonèení";

// Used in week.php
$vocab["weekbefore"]         = "Týden dozadu";
$vocab["weekafter"]          = "Týden dopøedu";
$vocab["gotothisweek"]       = "Tento týden";

// Used in month.php
$vocab["monthbefore"]        = "Mìsíc dozadu";
$vocab["monthafter"]         = "Mìsic dopøedu";
$vocab["gotothismonth"]      = "Tento mìsíc";

// Used in {day week month}.php
$vocab["no_rooms_for_area"]  = "Pro tuto místnost není definována žadná oblast!";

// Used in admin.php
$vocab["edit"]               = "Editovat";
$vocab["delete"]             = "Smazat";
$vocab["rooms"]              = "Místnosti";
$vocab["in"]                 = "v";
$vocab["noareas"]            = "Žádné oblasti";
$vocab["addarea"]            = "Pøidat oblast";
$vocab["name"]               = "Jméno";
$vocab["noarea"]             = "Není vybrána žádná oblast";
$vocab["browserlang"]        = "Prohlížec je nastaven k použití";
$vocab["addroom"]            = "Pøidat místnost";
$vocab["capacity"]           = "Kapacita";
$vocab["norooms"]            = "Žádná místnost.";
$vocab["administration"]     = "Administrace";

// Used in edit_area_room.php
$vocab["editarea"]           = "Editovat oblast";
$vocab["change"]             = "Zmìna";
$vocab["backadmin"]          = "Návrat do administrace";
$vocab["editroomarea"]       = "Editovat popis oblasti nebo místnosti";
$vocab["editroom"]           = "Editovat místnosti";
$vocab["update_room_failed"] = "Chyba editace místnosti: ";
$vocab["error_room"]         = "Chyba: místnost ";
$vocab["not_found"]          = " nenalezen";
$vocab["update_area_failed"] = "Chyba editace oblasti: ";
$vocab["error_area"]         = "Chyba: oblast ";
$vocab["room_admin_email"]   = "Email administrátora místnosti";
$vocab["area_admin_email"]   = "Email administrátora oblasti";
$vocab["invalid_email"]      = "Špatný email!";

// Used in del.php
$vocab["deletefollowing"]    = "Bylo smazáno rezervování";
$vocab["sure"]               = "Jste si jistý?";
$vocab["YES"]                = "ANO";
$vocab["NO"]                 = "NE";
$vocab["delarea"]            = "Musíte smazat všechny místnosti v této oblasti pøedtím než ji mùžete smazat<p>";

// Used in help.php
$vocab["about_mrbs"]         = "O MRBS";
$vocab["database"]           = "Databáze";
$vocab["system"]             = "Systém";
$vocab["please_contact"]     = "Prosím kontaktujte ";
$vocab["for_any_questions"]  = "pokud máte nìjaké další otázky.";

// Used in mysql.inc AND pgsql.inc
$vocab["failed_connect_db"]  = "Fatalní chyba: Nepodaøilo se pøipojit do databáze";

?>

File: web/lang.pl
<?php // -*-mode: PHP; coding:utf-8;-*-
// $Id$

// Ten plik zawiera kod PHP, który określa język konkretnych stringów.
// Domyślne stringi pochodzą z lang.en, i jakiekolwiek zmiany w konkretnych
// plikach lokalizacji nadpiszą domyślne stringi. To jest plik dla języka angielskiego US/UK.
//
//
//
//
// Ten plik to kod PHP i proszę traktować go jako taki.

// Uzyte w style.inc
$vocab["mrbs"]               = "Meeting Room Booking System";

// Uzyte w functions.inc
$vocab["report"]             = "Raport";
$vocab["admin"]              = "Administrator";
$vocab["help"]               = "Pomoc";
$vocab["search"]             = "Szukaj";
$vocab["not_php3"]           = "UWAGA: To najprawdopodobniej nie dziala z PHP3";

// Uzyte w day.php
$vocab["bookingsfor"]        = "Rezerwacja dla";
$vocab["bookingsforpost"]    = ""; // To idzie po dacie
$vocab["areas"]              = "Strefy";
$vocab["daybefore"]          = "Idź do dnia przed";
$vocab["dayafter"]           = "Idź do dnia po";
$vocab["gototoday"]          = "Idź do dnia dzisiejszego";
$vocab["goto"]               = "Idź do";
$vocab["highlight_line"]     = "Podświetl tą linie";
$vocab["click_to_reserve"]   = "Kliknij na komórkę aby zarezerwować.";

// Uzyte w trailer.inc
$vocab["viewday"]            = "Zobacz Dzień";
$vocab["viewweek"]           = "Zobacz Tydzień";
$vocab["viewmonth"]          = "Zobacz Miesiąc";
$vocab["ppreview"]           = "Wydrukuj Podgląd";

// Uzyte w edit_entry.php
$vocab["addentry"]           = "Dodaj Wpis";
$vocab["editentry"]          = "Edytuj Wpis";
$vocab["copyentry"]          = "Kopiuj Wpis";
$vocab["editseries"]         = "Edytuj Serie";
$vocab["copyseries"]         = "Kopiuj Serie";
$vocab["namebooker"]         = "Krótki Opis";
$vocab["fulldescription"]    = "Pełen Opis:<br>&nbsp;&nbsp;(Liczba osób,<br>&nbsp;&nbsp;Wewnętrzne/Zewnętrzne itp)";
$vocab["date"]               = "Data";
$vocab["start_date"]         = "Czas Rozpoczęcia";
$vocab["end_date"]           = "Czas Edycji";
$vocab["time"]               = "Czas";
$vocab["period"]             = "Okres";
$vocab["duration"]           = "Czas Trwania";
$vocab["seconds"]            = "sekundy";
$vocab["minutes"]            = "minuty";
$vocab["hours"]              = "godziny";
$vocab["days"]               = "dni";
$vocab["weeks"]              = "tygodnie";
$vocab["years"]              = "lata";
$vocab["periods"]            = "okresy";
$vocab["all_day"]            = "Cały dzień";
$vocab["type"]               = "Typ";
$vocab["internal"]           = "Wewnętrzny";
$vocab["external"]           = "Zewnętrzny";
$vocab["save"]               = "Zapisz";
$vocab["rep_type"]           = "Powtórz Typ";
$vocab["rep_type_0"]         = "Żaden";
$vocab["rep_type_1"]         = "Dzienny";
$vocab["rep_type_2"]         = "Tygodniowy";
$vocab["rep_type_3"]         = "Miesięczny";
$vocab["rep_type_4"]         = "Roczny";
$vocab["rep_type_5"]         = "Miesięczny, odpowiadający dzień";
$vocab["rep_type_6"]         = "n-Tygodniowy";
$vocab["rep_end_date"]       = "Powtórz Datę końcową";
$vocab["rep_rep_day"]        = "Powtórz Datę";
$vocab["rep_for_weekly"]     = "(dla (n-)tygodniowy)";
$vocab["rep_freq"]           = "Częstotliwość";
$vocab["rep_num_weeks"]      = "Liczba Tygodni";
$vocab["rep_for_nweekly"]    = "(dla n-tygodniowy)";
$vocab["ctrl_click"]         = "Klikaj myszą przytrzymując klawisz Ctrl aby wybrać więcej niż jeden pokój";
$vocab["entryid"]            = "ID Wpisu ";
$vocab["repeat_id"]          = "Powtórz ID ";
$vocab["you_have_not_entered"] = "Nie wprowadziłeś";
$vocab["you_have_not_selected"] = "Nie wybrałeś";
$vocab["valid_room"]         = "pokój.";
$vocab["brief_description"]  = "Krótki Opis.";
$vocab["useful_n-weekly_value"] = "przydatna n-tygodniowa wartość.";

// Uzyte w view_entry.php
$vocab["description"]        = "Opis";
$vocab["room"]               = "Pokój";
$vocab["createdby"]          = "Utworzony Przez";
$vocab["lastupdate"]         = "Ostatnio Aktualizowany";
$vocab["deleteentry"]        = "Usuń Wpis";
$vocab["deleteseries"]       = "Usuń Wpisy";
$vocab["confirmdel"]         = "Czy jesteś pewien\\nże chcesz usunąć\\nten wpis?\\n\\n";
$vocab["returnprev"]         = "Powrót do poprzedniej strony";
$vocab["invalid_entry_id"]   = "Niewłaściwy ID wpisu";
$vocab["invalid_series_id"]  = "Niewłaściwy ID serii";

// Uzyte w edit_entry_handler.php
$vocab["error"]              = "Błąd";
$vocab["sched_conflict"]     = "Konflikt Planowania";
$vocab["conflict"]           = "Nowa rezerwacja będzie kolidowała z następującym(i) wpisem(-ami)";
$vocab["too_may_entrys"]     = "Wybrana opcja utworzy zbyt wiele wpisów.<br>Proszę użyć innego wyboru!";
$vocab["returncal"]          = "Powrót do widoku kalendarza";
$vocab["failed_to_acquire"]  = "Błąd podczas uzyskania wyłącznego dostępu do bazy.";
$vocab["invalid_booking"]    = "Niewłaściwa rezerwacja";
$vocab["must_set_description"] = "Musisz podać krótki opis rezerwacji. Proszę, wróć i wprowadź go.";
$vocab["mail_subject_new_entry"]     = "Wpis wprowadzony w $mrbs_company MRBS";
$vocab["mail_subject_changed_entry"] = "Wpis zmieniony w $mrbs_company MRBS";
$vocab["mail_subject_delete"]        = "Wpis usunięty z $mrbs_company MRBS";
$vocab["mail_body_new_entry"]     = "Wpis został utworzony, oto szczegóły operacji:";
$vocab["mail_body_changed_entry"] = "Wpis został zmieniony, oto szczegóły operacji:";
$vocab["mail_body_del_entry"]     = "Wpis został usunięty, oto szczegóły operacji:";

// Autentykacja
$vocab["accessdenied"]       = "Dostęp Wzbroniony";
$vocab["norights"]           = "Nie masz uprawnień do zmiany tego obiektu.";
$vocab["please_login"]       = "Proszę, zaloguj się";
$vocab["users.name"]          = "Nazwa";
$vocab["users.password"]      = "Hasło";
$vocab["users.level"]         = "Uprawnienia";
$vocab["unknown_user"]       = "Nieznany użytkownik";
$vocab["you_are"]            = "Zalogowany";
$vocab["login"]              = "Zaloguj";
$vocab["logoff"]             = "Wyloguj";

// Autentykacja - baza
$vocab["user_list"]          = "Lista użytkowników";
$vocab["edit_user"]          = "Edytuj";
$vocab["delete_user"]        = "Usuń";
//$vocab["users.name"]         = Use the same as above, for consistency.
//$vocab["users.password"]     = Use the same as above, for consistency.
$vocab["users.email"]         = "adres email";
$vocab["password_twice"]     = "Jeśli chcesz zmienić hasło, wpisz dwukrotnie nowe hasło";
$vocab["passwords_not_eq"]   = "Błąd! Powtórzone hasła różnią się";
$vocab["add_new_user"]       = "Dodaj nowego użytkownika";
$vocab["action"]             = "Akcja";
$vocab["user"]               = "Użytkownik";
$vocab["administrator"]      = "Administrator";
$vocab["unknown"]            = "Nieznany";
$vocab["ok"]                 = "OK";
$vocab["show_my_entries"]    = "Wyświetl moje wszystkie nadchodzące wpisy";
$vocab["no_users_initial"]   = "Brak użytkownika w bazie, możliwe utworzenie początkowego użytkownika";
$vocab["no_users_create_first_admin"] = "Utwórz użytkownika z uprawnieniami administratora. Potem logując się na niego, utworzysz pozostałych użytkowników.";

// Uzyte w search.php
$vocab["invalid_search"]     = "Pusty lub niewłaściwy łańcuch wyszukiwania.";
$vocab["search_results"]     = "Wyniki Wyszukiwania dla";
$vocab["nothing_found"]      = "Brak wyników spełniających kryteria.";
$vocab["records"]            = "Wyniki ";
$vocab["through"]            = " przez ";
$vocab["of"]                 = " z ";
$vocab["previous"]           = "Poprzedni";
$vocab["next"]               = "Następny";
$vocab["entry"]              = "Wpis";
$vocab["view"]               = "Widok";
$vocab["advanced_search"]    = "Wyszukiwanie zaawansowane";
$vocab["search_button"]      = "Wyszukiwanie";
$vocab["search_for"]         = "Szukaj";
$vocab["from"]               = "Od";

// Uzyte w report.php
$vocab["report_on"]          = "Raport ze Zpotkań";
$vocab["report_start"]       = "Data początku raportu";
$vocab["report_end"]         = "Data końca raportu";
$vocab["match_area"]         = "Zgodny Teren";
$vocab["match_room"]         = "Zgodny Pokój";
$vocab["match_type"]         = "Zgodny Typ";
$vocab["ctrl_click_type"]    = "Klikaj myszą przytrzymując klawisz Ctrl aby wybrać więcej opcji";
$vocab["match_entry"]        = "Zgodny krótki opis";
$vocab["match_descr"]        = "Zgodny pełny opis";
$vocab["include"]            = "Dołącz";
$vocab["report_only"]        = "Tylko Raport";
$vocab["summary_only"]       = "Tylko Podsumowanie";
$vocab["report_and_summary"] = "Raport i Podsumowanie";
$vocab["summarize_by"]       = "Sumowanie po";
$vocab["sum_by_descrip"]     = "Krótki opis";
$vocab["sum_by_creator"]     = "Autor";
$vocab["entry_found"]        = "Pozycja Znaleziona";
$vocab["entries_found"]      = "Pozycje Znaleziono";
$vocab["summary_header"]     = "Podsumowanie Godzin (Pozycje)";
$vocab["summary_header_per"] = "Podsumowanie Okresów (Pozycje)";
$vocab["total"]              = "Razem";
$vocab["submitquery"]        = "Uruchom Raport";
$vocab["sort_rep"]           = "Sortuj Raport po";
$vocab["sort_rep_time"]      = "Start Data/Czas";
$vocab["rep_dsp"]            = "Wyświetl w raporcie";
$vocab["rep_dsp_dur"]        = "Trwanie";
$vocab["rep_dsp_end"]        = "Czas Zakończenia";

// Uzyte w week.php
$vocab["weekbefore"]         = "Przejdź do Poprzedniego Tygodnia";
$vocab["weekafter"]          = "Przejdź do Następnego Tygodnia";
$vocab["gotothisweek"]       = "Przejdź Do Bieżącego Tygodnia";

// Uzyte w month.php
$vocab["monthbefore"]        = "Przejdź do Poprzedniego Miesiąca";
$vocab["monthafter"]         = "Przejdż do Następnego Miesiąca";
$vocab["gotothismonth"]      = "Przejdź do Bieżącegoo Miesiąca";

// Uzyte w {day week month}.php
$vocab["no_rooms_for_area"]  = "Brak zdefiniowanych pokoi dla tej strefy";

// Uzyte w admin.php
$vocab["edit"]               = "Edycja";
$vocab["delete"]             = "Usuń";
$vocab["rooms"]              = "Pokoje";
$vocab["in"]                 = "w";
$vocab["noareas"]            = "Brak Stref";
$vocab["addarea"]            = "Dodaj Strefę";
$vocab["name"]               = "Nazwa";
$vocab["noarea"]             = "Brak wybranych stref";
$vocab["browserlang"]        = "Twoja przeglądarka jest ustwaiona na następującą kolejność preferencji językowych";
$vocab["addroom"]            = "Dodaj Pokój";
$vocab["capacity"]           = "Pojemność";
$vocab["norooms"]            = "Brak Pokoi.";
$vocab["administration"]     = "Administracja";

// Uzyte w edit_area_room.php
$vocab["editarea"]           = "Edytuj Strefę";
$vocab["change"]             = "Zmień";
$vocab["backadmin"]          = "Powrót do Admin";
$vocab["editroomarea"]       = "Edycja Strefy lub Opis Pokoju";
$vocab["editroom"]           = "Edycja Pokoju";
$vocab["update_room_failed"] = "Uaktualnij pole pokoju: ";
$vocab["error_room"]         = "Błąd: Pokój ";
$vocab["not_found"]          = " nie znaleziono";
$vocab["update_area_failed"] = "Uaktualnij pole strefy: ";
$vocab["error_area"]         = "Błąd: strefa ";
$vocab["room_admin_email"]   = "Email do administratora pokoju";
$vocab["area_admin_email"]   = "Email do administratora strefy";
$vocab["invalid_email"]      = "Błędny email!";

// Uzyte w edit_users.php
$vocab["name_empty"]         = "Musisz podać nazwę.";
$vocab["name_not_unique"]    = "Nazwa aktualnie istnieje.  Prosze wybrać inną.";

// Uzyte w del.php
$vocab["deletefollowing"]    = "To skasuje aktualne rezerwacje";
$vocab["sure"]               = "Czy aby na pewno?";
$vocab["YES"]                = "TAK";
$vocab["NO"]                 = "NIE";
$vocab["delarea"]            = "Musisz usunąc wszystkie pokoje w tej strefie przed usunięciem jej<p>";

// Uzyte w help.php
$vocab["about_mrbs"]         = "Informacje o MRBS";
$vocab["database"]           = "Baza Danch";
$vocab["system"]             = "System";
$vocab["servertime"]         = "Czas Serwera";
$vocab["please_contact"]     = "Prosze o kontakt ";
$vocab["for_any_questions"]  = "W sprawie pytań na które brak tu odpowiedzi.";

// Uzyte w mysql.inc AND pgsql.inc
$vocab["failed_connect_db"]  = "Błąd Krytyczny: Błąd połączenia z bazą danych";
